index.d.ts

```diff
@@ -899,7 +899,7 @@ declare namespace Eris {
     ): T;
     (event: "voiceStateUpdate", listener: (member: Member, oldState: OldVoiceState) => void): T;
     (event: "warn" | "debug", listener: (message: string, id: number) => void): T;
-    (event: string, listener: Function): T;
+    (event: string, listener: (...args: any[]) => void): T;
```

Comment:
```suggestion
    (event: string, listener: (...args: unknown[]) => void): T;
```

Reply:
`any` is actually consistent with the events typings